src/QuickMarcEditor/QuickMarcEditorContainer.test.js

```diff
@@ -174,7 +170,7 @@ describe('Given Quick Marc Editor Container', () => {
         });
       });
 
-      expect(mockFetchLinksCount).toHaveBeenCalledWith([match.params.externalId]);
```

Comment:
Delete `mockFetchLinksCount` declaration on line 27 since this variable isn't used anywhere else now (see Github Check)